ledger-core/v2/Makefile.testing

```diff
@@ -26,8 +26,9 @@ test_func: ## run functest FUNCTEST_COUNT times
 	CGO_ENABLED=1 $(GOTEST) -test.v $(TEST_ARGS) -tags "functest" ./application/functest -count=$(FUNCTEST_COUNT)
 
 .PHONY: test_func_slow
-test_func_slow: ## run functest 100 times with enabled pulse 3 sec
-	FUNCTEST_COUNT=100 PULSAR_ONESHOT=FALSE PULSAR_PULSETIME=3000 TEST_ARGS='-timeout 1200s -failfast' make test_func
+test_func_slow: # Run functests as long as practical to catch unexpected behavior during pulse change
+	pgrep insolard | xargs kill -9 # dirty hack to terminate all insolard processes
+	FUNCTEST_COUNT=100 PULSAR_ONESHOT=FALSE PULSAR_PULSETIME=5000 TEST_ARGS='-timeout 1200s -failfast' make test_func
```

Comment:
I guess that PULSAR_ONESHOT should be true. When func tests will run better on multiple pulses, then will enable them for both func and func_slow

Reply:
PULSAR_ONESHOT=$(PULSAR_ONESHOT) should do the work

Reply:
I'm very sorry, bit this strongly contradicts the Jira task.